README.md

```diff
@@ -5,6 +5,14 @@ Docker images for deploying and running the Confluent Platform.  The images are
 
 Full documentation for using the images can be found [here](http://docs.confluent.io/current/cp-docker-images/docs/intro.html).
 
+# Deprecation Notice
+
+This is used for building images for version 4.1.x or lower, and should not be used for adding new images. If any new
+images is added, please make sure to add to our [Release steps](https://confluentinc.atlassian.net/wiki/x/ywE6Cg) as
```

Comment:
fyi, this is a public repo, not sure if it makes sense to add private links in here.

Reply:
+1 can we remove this link @maxzheng ?